tamagotchi-01/interface/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+#![no_std]
+
+use codec::{Decode, Encode};
+use gmeta::{InOut, Metadata};
+use gstd::{prelude::*};
+use scale_info::TypeInfo;
+pub struct ProgramMetadata;
+
+impl Metadata for ProgramMetadata {
+    type Init = InOut<String,String>;
```

Comment:
As we don't reply from the `init` function, we are to set this type to `In`.
```suggestion
    type Init = In<String>;
```